src/components/Search.js

```diff
@@ -12,11 +12,13 @@ export default class Search extends React.Component {
   }
 
   onChange(e) {
-    var value = e.target.value;
+    const value = e.target.value,
+          keywords = e.target.value.toLowerCase().split(' ');
+
     this.props.onFilteredData(_.filter(this.props.data, function (row) {
-      return _.find(_.values(row), function (val) {
-        return (val.toString().toLowerCase()).indexOf(value.toLowerCase()) !== -1;
-      });
+      const values = _.values(row);
+      const words = _.join(values, ' ').toLowerCase();
```

Comment:
https://jsbin.com/vesowu/edit?js,console

i can't test it right now, but that's my understanding of how `_.includes` works? it would just do exact matches, not substring matches.

also you might want to `toLowerCase` the `keywords`, just in case someone types 'Florida'